tapestry.js

```diff
@@ -1733,8 +1756,8 @@ function setupLightbox(id, mediaFormat, mediaType, mediaUrl, width, height) {
         media.on(loadEvent, function() {
             changeToViewMode(lightboxDimensions);
             window.setTimeout(function(){
-                height = $('#spotlight-content > *').outerHeight();
-                width = $('#spotlight-content > *').outerWidth();
+                height = $('#spotlight-content').outerHeight();
```

Comment:
This doesn't work. The height of the spotlight-content is currently clipping the elements inside, so this code used to increase its height to fit everything and now it's not doing that. You'll need to find the total height of all the elements inside or otherwise use another method to set the height of the container to fit everything. Remember that the description may be more than 1 line.

Reply:
Reverted back.